.changeset/proud-adults-bathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-ldap': minor
```

Comment:
Is this actually a `minor` release? Think that it shoud be a `patch` release as it's not a breaking change to the API, and we're below `<1.x` for now?

Reply:
Ah good call, I'll drop to a patch